### PR TITLE
Patch Release for 7.1.9

### PR DESCRIPTION
**Pull request by rrbadiani**
# Patch Release

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,17 @@ Ansible Playbooks for Confluent Platform - Release Notes
 
 .. contents:: Topics
 
+v7.1.9
+======
+
+Notable enhancements
+-------------
+
+- Updated confluent cli version to 2.38.1
```
Comment (mansisinha) [inline CHANGELOG.rst:13]: default confluent cli

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,17 @@ Ansible Playbooks for Confluent Platform - Release Notes
...
+- Fixed bugs in discovery to generate inventory file with appropriate security protocols
```
Comment (mansisinha) [inline CHANGELOG.rst:14]: discovery code is not present with 7.1. it was introduced in 7.4, please remove this from all versions lower than 7.4

Comment (rrbadiani) [reply CHANGELOG.rst]: Done

File: CHANGELOG.rst (hunk 1 above)
Comment (rrbadiani) [reply CHANGELOG.rst]: Done

**Review by mansisinha — APPROVED** [review]

**Pull request merged**